Indexing/LiteralIndex.cpp

```diff
@@ -107,7 +107,11 @@ void UnitClauseLiteralIndex::handleClause(Clause* c, bool adding)
 {
   if(c->length()==1) {
     TIME_TRACE("unit clause index maintenance");
-    
+
+    if (_skipNonequational && !(*c)[0]->isEquality()) {
```

Comment:
This seems like something other indices might use, right? Did e.g. LHS indices get updated?

Reply:
LHS is equational only by design, isn't it?

Reply:
Right - but if indices can inherit from this and get the right behaviour that would be a nice pattern. Looking at it maybe it doesn't link up the way I think it does, sorry.